content/docs/integrations/github-discovery/index.md

```diff
@@ -82,11 +85,10 @@ In this example, Roadie will load any `.yaml` file inside a root level `.roadie`
 
 Add a Backstage component file to a repository which matches the configured discovery url. After committing it to the correct branch, you should see it almost immediately in your catalog.
 
-
 ## Decorating Catalog Entities
 
 You can add the primary language used by a repository as a tag or label to the Roadie Backstage entity without updating your yaml files by turning on the decoration feature in `/administration/settings/catalog`.
 
-There will be a lag of a round 10 minutes for all entities to be tagged, and when turning it off, there will be a similar lag for the tags to be removed. 
+There will be a lag of a round 10 minutes for all entities to be tagged, and when turning it off, there will be a similar lag for the tags to be removed.
```

Comment:
```suggestion
It will take approximately 10 minutes for all entities to be tagged. Similarly, when disabling this feature, it will take approximately 10 minutes for the tags to be removed.
```